src/Management/src/Endpoint/ManagementPort/ManagementPortMiddleware.cs

```diff
@@ -52,18 +54,47 @@ public async Task InvokeAsync(HttpContext context)
         }
     }
 
-    private static bool IsRequestAllowed(HttpRequest request, ManagementOptions managementOptions)
+    private bool IsRequestAllowed(HttpRequest request, ManagementOptions managementOptions)
     {
-        if (int.TryParse(managementOptions.Port, CultureInfo.InvariantCulture, out int managementPort) && managementPort > 0)
+        if (!int.TryParse(managementOptions.Port, CultureInfo.InvariantCulture, out int managementPort) || managementPort <= 0)
         {
-            bool isManagementPath = request.Path.StartsWithSegments(managementOptions.Path);
-            bool isManagementScheme = managementOptions.SslEnabled ? request.Scheme == Uri.UriSchemeHttps : request.Scheme == Uri.UriSchemeHttp;
-            bool isManagementPort = request.Host.Port == managementPort;
+            return true;
+        }
+
+        bool isManagementPath = request.Path.StartsWithSegments(managementOptions.Path);
+        bool isManagementScheme = managementOptions.SslEnabled ? request.Scheme == Uri.UriSchemeHttps : request.Scheme == Uri.UriSchemeHttp;
+        bool isManagementPort = request.Host.Port == managementPort;
+
+        string? instancePorts = Environment.GetEnvironmentVariable("CF_INSTANCE_PORTS");
```

Comment:
Just wanted to point out we have various `CF_*` environment variables exposed in `CloudFoundryApplicationOptions` at https://github.com/SteeltoeOSS/Steeltoe/blob/bee80fcd69aabea12218d9c0fc1f870b72479109/src/Configuration/src/CloudFoundry/CloudFoundryApplicationOptions.cs#L98.

We could consider adding `CF_INSTANCE_PORTS` there. And then we could consider taking a dependency on `CloudFoundryApplicationOptions`.

On the other hand, there are more entries in the codebase that are missing in `CloudFoundryApplicationOptions` already: `CF_SYSTEM_CERT_PATH`, `CF_INSTANCE_CERT`, `CF_INSTANCE_KEY`, so it may not be worth the effort.

Reply:
I do generally like idea of making `CloudFoundryApplicationOptions ` more useful and putting it there, but also this is the first time I've needed to use it in 7+ years